jsconfig.json

```diff
@@ -0,0 +1 @@
+{"include": ["frontend/src/**", "enterprise/frontent/src/**"]}
```

Comment:
This is not specific to this fix but by adding it I was able to get jump-to-definition, documentation, and auto-complete working in Emacs with tide-mode